app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: None) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError()
+
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError()

```

Comment:
Consider providing a descriptive error message when raising `NotWearingMaskError` to make it clear why the error is being raised.